e2e/features/edit-case-listing.feature

```diff
@@ -0,0 +1,75 @@
+    And I add the appeal response
+    And I request hearing requirements
+    And I switch to be a `Admin Officer`
+    And I list the case
+    And I switch to be a `Case Officer`
+
+  @edit-case-listing @RIA-362
+  Scenario: Edit the case listing
+
+    When I select the `Edit case listing` Next step
+    Then I am on the `Edit case listing` page
+    And I should see the option `Manchester` for the `Hearing centre` field
+    And I should see the option `Taylor House` for the `Hearing centre` field
+    And I should see the option `30 minutes` for the `Length of hearing` field
+    And I should see the option `1 hour` for the `Length of hearing` field
+    And I should see the option `1 hour 30 minutes` for the `Length of hearing` field
+    And I should see the option `2 hours` for the `Length of hearing` field
+    And I should see the option `2 hours 30 minutes` for the `Length of hearing` field
+    And I should see the option `3 hours` for the `Length of hearing` field
+    And I should see the option `3 hours 30 minutes` for the `Length of hearing` field
+    And I should see the option `4 hours` for the `Length of hearing` field
+    And I should see the option `4 hours 30 minutes` for the `Length of hearing` field
+    And I should see the option `5 hours` for the `Length of hearing` field
+    And I should see the option `5 hours 30 minutes` for the `Length of hearing` field
+    And I should see the option `6 hours` for the `Length of hearing` field
+
+    When I select `6 hours` for the `Length of hearing` field
+    And I select `Manchester` for the `Hearing centre` field
+    And I select `{$TODAY+14|DD-MM-YYYY} 10:30:00` for the `Hearing date and time` field
+    And I click the `Continue` button
+
+    When I type `Personal vulnerabilities...` for the `Adjustments to accommodate vulnerabilities (Optional)` field
+    And I type `Multimedia equipment...` for the `Multimedia equipment (Optional)` field
+    And I type `Single sex court...` for the `Single-sex court (Optional)` field
+    And I type `In camera court...` for the `In camera court (Optional)` field
+    And I type `Other requests...` for the `Other adjustments (Optional)` field
+    And I click the `Continue` button
+
+    Then I am on the `Check your answers` page
+    And I should see `Manchester` for the `Hearing centre` field
+    And I should see `6 hours` for the `Length of hearing` field
+    And I should see `{$TODAY+14|D MMM YYYY}, 10:30:00 AM` for the `Hearing date and time` field
+
+    When I click the `List case` button
+    Then I should see the text `The case has been re-listed`
+    And I should see the text `What happens next`
+    And I should see the text `A new hearing notice has been generated. All parties will be notified by email.`
+
+    When I click the `Close and Return to case details` button
+    Then I should see an alert confirming the case `has been updated with event: Edit case listing`
+    And I see the open case
+
```

Comment:
I think it's worthing assert the Hearing Details on overview Tab. Because the Hearing Details has changed..